__tests__/posts.ts

```diff
@@ -3554,20 +3709,44 @@ describe('mutation votePost', () => {
     });
     const post = await con.getRepository(Post).findOneBy({ id: 'p1' });
     expect(post?.downvotes).toEqual(4);
+  };
+
+  it('should downvote and NOT update source flags upvotes count', async () => {
+    loggedUser = '1';
+    const source = await con.getRepository(Source).findOneByOrFail({ id: 'a' });
+    expect(source.flags.totalUpvotes).toEqual(undefined);
+
+    await testDownvote();
+
+    // should not be affected since this is not a squad
+    const updatedSource = await con
+      .getRepository(Source)
+      .findOneByOrFail({ id: 'a' });
+    expect(updatedSource.flags.totalUpvotes).toEqual(undefined);
   });
 
-  it('should cancel vote', async () => {
```

Comment:
Can we keep this test as is and then just add new tests after. Old tests should pass.

Reply:
Yeah, it is probably better to keep old ones untouched.